src/product-switch.scss

```diff
@@ -57,6 +57,10 @@ $block: #{$fd-namespace}-product-switch;
     }
 
     width: 47rem;
+
+    > li {
+      list-style-type: none;
```

Comment:
Could you add this rule to `&__item` too?

Reply:
If we do this why don't we just target all li under list and make them list-style-type:none